pylintrc

```diff
@@ -146,6 +146,52 @@ disable=abstract-method,
         wrong-import-order,
         xrange-builtin,
         zip-builtin-not-iterating,
+        no-self-argument,
```

Comment:
do not change pylintrc. we may just allow pylint to fail, or ignore the fail status in workflow

Reply:
Done. reverted those changes